documentation/docs/Exercises/Exercise 5.md

```diff
@@ -18,7 +18,7 @@ Axelar Network is a work in progress. At no point in time should you transfer an
 ## Prerequisites
 - Complete all steps from [Setup with Docker](/setup-docker) or [Setup with Binaries](/setup-binaries)
 - Have a Ethereum wallet setup and have an Ethereum address funded with some Ether (You can also choose to use the [Chrome plugin](https://chrome.google.com/webstore/detail/mew-cx/nlbmnnijcnlegkjjpcfjclmcfggfefdm?hl=en))
-- Your Axelar Network address have`satoshi` and `uphoton (ibc/287EE075B7AADDEB240AFE74FA2108CDACA50A7CCD013FA4C1FCD142AFA9CA9A)` balances. If you do not have these assets, please follow `Exercise 3` and `Exercise 4` to mint some.
+- Your Axelar Network address has a positive `uaxl` balance. You can get AXL tokens from the [Axelar faucet](http://faucet.testnet.axelar.dev/).
```

Comment:
positive sounds weird to me. Maybe "You have some uaxl tokens on your Axelar Network address" or something similar?